Frends.Community.Sftp.Tests/Frends.Community.Sftp.Tests.cs

```diff
@@ -2,69 +2,69 @@
 using System.Collections.Generic;

```

Comment:
https://github.com/CommunityHiQ/Frends.Community.Sftp/blob/33f966138d1c7e6d9f4d8e30d730f3e930de2f0e/Frends.Community.Sftp.Tests/Frends.Community.Sftp.Tests.cs#L1
You are not using this namespace, so it can be removed.